packages/mui-base/src/ModalUnstyled/ModalUnstyled.d.ts

```diff
@@ -152,11 +152,12 @@ export type ExtendModalUnstyled<M extends OverridableTypeMap> = OverridableCompo
  *
  * Demos:
  *
- * - [Modal](https://mui.com/components/modal/)
+ * - [Modal](https://mui.com/base/react-modal/)
+ * - [Modal](https://mui.com/material/react-modal/)
```

Comment:
@siriwatknp should this link only to the base page? There are no more demos on the material page.